templates/confidential-computing/index.html

```diff
@@ -217,10 +217,10 @@ <h2>Launch enterprise-grade confidential VMs on all major public clouds</h2>
           <div class="row">
             <div class="col-2 col-medium-3 col-small-2 u-pad-logo">
```

Comment:
These should all be col-3, this is what's causing it to look extra small on desktop when you try to change the dimensions 
```suggestion
            <div class="col-3 col-medium-3 col-small-2 u-pad-logo">
```